### DISCUSSION TITLE
Add a global plug-in system

### DISCUSSION THREAD

**Discussion by L2750558108** [godotengine/godot-proposals#6023] [Engine Core]
When creating a new project, it is always necessary to re-import commonly used plugins, and it may be better if they can be installed into the editor and enabled in each project

**Comment by Hiiamwilliam** [comment]
There's a proposal for that: https://github.com/godotengine/godot-proposals/issues/831

**Reply by L2750558108** [reply]
Can I ask when it might be implemented? It's been 2 years since that post...

**Reply by Zireael07** [reply]
You can ask but you won't get an answer - Godot is done on a best effort basis, there are never any ETAs
